### PR TITLE
add option to show cache age ( AB#15927 )

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: run_lmod_cache.py [hunk 1]
```diff
@@ -121,7 +122,18 @@ def main():
...
         'module-basedir': ('Specify the base dir for the modules', 'str', 'store', MODULES_BASEDIR),
+        'check-cache-age': ('Show age in minutes of oldest cache and exit', None, 'store_true', False),
     }
+    opts = ExtendedSimpleOption(options, run_prologue=False)
+
+    timestamp = find_oldest_cache(opts.options.module_basedir, archs=opts.options.architecture)
+    age = time.time() - timestamp
+
+    if opts.options.check_cache_age:
+        print(int(age/60.0))
```
Comment (wpoely86) [inline run_lmod_cache.py]: for zabbix it's easier if it's in seconds

File: run_lmod_cache.py [hunk 2]
```diff
@@ -121,7 +122,18 @@ def main():
         'freshness-threshold': ('The interval in minutes for how long we consider the cache to be fresh',
                                 'int', 'store', 60*4),  # cron runs every 3 hours
         'module-basedir': ('Specify the base dir for the modules', 'str', 'store', MODULES_BASEDIR),
+        'check-cache-age': ('Show age in minutes of oldest cache and exit', None, 'store_true', False),
     }
+    opts = ExtendedSimpleOption(options, run_prologue=False)
```
Comment (wpoely86) [inline run_lmod_cache.py]: why are you adding this again?

File: run_lmod_cache.py [hunk 3]
```diff
@@ -133,10 +145,8 @@ def main():
                 opts.critical("Lmod cache update failed")
 
         opts.log.info("Checking the Lmod cache freshness")
-        timestamp = find_oldest_cache(opts.options.module_basedir, archs=opts.options.architecture)
-
-        # give a warning when the cache is older then --freshness-threshold
-        if (time.time() - timestamp) > opts.options.freshness_threshold * 60:
```
Comment (wpoely86) [inline run_lmod_cache.py:139]: now I look at the code, we actually already have this more or less? We only need to run with `--disable-create-cache'?

Comment (smoors) [reply run_lmod_cache.py:139]: for zabbix it's easier if we just send a number (the age)

File: run_lmod_cache.py (hunk 2 above)
Comment (smoors) [reply run_lmod_cache.py]: replaced with SimpleOption to avoid running this twice

**Review by wpoely86 — APPROVED** [review]

**Pull request merged**